tools/StaticAnalysis/Program.cs

```diff
@@ -49,7 +49,7 @@ public static void Main(string[] args)
             Console.WriteLine("Analyzer invoked with parameters: {0}", string.Join(" ", args));
             try
             {
-                string installDir = null;
+                string installDir = "C:\\Users\\yunwang\\source\\repos\\azure-powershell\\artifacts\\Debug";
```

Comment:
This has a reference to a specific home directory & isn't generic.

Reply:
Hi @Skuldo, thanks for your contribution. This is just a draft PR. But I will fix this when other logic is completed.